### PR TITLE
128 design notification page

### PR DESCRIPTION
**Pull request by kphutiyagae**
# Description

Added Notifications page angular functionality (without NGXS for now)

Fixes # 128 design notification page

## Type of change

Feature

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

- [x] Unit tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: libs/app/notifications/feature/src/notifications.page.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<div class="h-full w-full">
```
Comment (ryanbasiltrickett) [inline libs/app/notifications/feature/src/notifications.page.html:1]: Ensure the background colour is the same as for the application and allow scrollable incase it over flows

File: libs/app/notifications/feature/src/notifications.page.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  <div class="h-1/5 w-full flex flex-row items-center">
```
Comment (ryanbasiltrickett) [inline libs/app/notifications/feature/src/notifications.page.html:2]: Add back button

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]
Please revise

File: libs/app/notifications/feature/src/notifications.page.html (hunk 2 above)
Comment (kphutiyagae) [reply libs/app/notifications/feature/src/notifications.page.html]: Done.

File: libs/app/notifications/feature/src/notifications.page.html (hunk 1 above)
Comment (kphutiyagae) [reply libs/app/notifications/feature/src/notifications.page.html]: Done

**Review by ryanbasiltrickett — APPROVED** [review]

**Pull request merged**